explorer/rest/db/DatabaseConnection.py

```diff
@@ -0,0 +1,37 @@
+from psycopg2 import pool
+
+
+class DatabaseConnection:
+	def __init__(self, db_config, min_connections=1, max_connections=10):
+		self.db_config = db_config
+		self.min_connections = min_connections
+		self.max_connections = max_connections
+		self.pool = None
+		self.connection = None
+
+	def connect(self):
+		"""Create connection pool."""
+
+		self.pool = pool.SimpleConnectionPool(
+			self.min_connections,
+			self.max_connections,
+			database=self.db_config['database'],
+			user=self.db_config['user'],
+			password=self.db_config['password'],
+			host=self.db_config['host'],
+			port=self.db_config['port']
```

Comment:
I have INI in my config.

```
[nem_db]
database = nem_db
user = user
password = xxx
host = xxx
port = xxx
```
When reading config it convert to dict instead of namedtuple.
https://docs.python.org/3/library/configparser.html

```
app.config.from_envvar('EXPLORER_REST_SETTINGS')

config = configparser.ConfigParser() # because of this
```

Reply:
aware, but you can create the named tuple from the dict and pass in the former